gradle.properties

```diff
@@ -28,13 +28,13 @@ artifactoryContextUrl = https://software.r3.com/artifactory
 # dokka need more metaspace - https://github.com/Kotlin/dokka/issues/1405
 org.gradle.jvmargs=-XX:MaxMetaspaceSize=2g
 internalPublishVersion = 1.+
-dokkaVersion = 1.6.+
-detektPluginVersion = 1.19.+
+dokkaVersion = 1.7.+
+detektPluginVersion = 1.20.+
 dependencyCheckVersion=0.42.+
 artifactoryPluginVersion = 4.28.2
 
 # Logging
-slf4jVersion = 1.7.32
+slf4jVersion = 1.7.36
```

Comment:
Note I am using the 2.0.0 series at home and there are some stupid warnings about non-implemented methods when linked to latest log4j, So I think it is correct to stay on the 1.7 series